.github/workflows/docker.yml

```diff
@@ -2,37 +2,15 @@ name: Build & Push
 #  Build & Push rebuilds the tendermint docker image on every push to master and creation of tags
 # and pushes the image to https://hub.docker.com/r/interchainio/simapp/tags
 on:
-  pull_request:
-  push:
-    branches:
-      - master
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
```

Comment:
why are we removing this logic?

Reply:
To make it consistent with our other repos and how we version and release other components. If you want an image built on every push or tag that is fine as well, just let me know.